### PR TITLE
Added table_screen_cell template

### PR DESCRIPTION
**Pull request by twerth**

### CODE REVIEW

File: bin/potion [hunk 1]
```diff
@@ -18,6 +18,7 @@ class PotionCommandLine
      > potion create model foo
      > potion create screen foo
      > potion create table_screen foo
+     > potion create table_screen_cell bar_cell
```
Comment (squidpunch) [inline bin/potion]: Can we just use 'cell' for the generator?  I'm a lazy typer :)

Comment (markrickert) [inline bin/potion]: What happens when we want to generate a collection view cell then?

Comment (twerth) [inline bin/potion]: Well, there is a metal cell too, was my thinking. But either way is cool with me.

Comment (squidpunch) [inline bin/potion]: Good point mark, verbosity wins out.

**Comment by twerth** [comment]
This is done, tested, and documented.

File: bin/potion (hunk 1 above)
Comment (markrickert) [inline bin/potion]: article-2539918-1aaf4f1100000578-431_634x515

**Pull request merged**